### DISCUSSION TITLE
AAIDrive not working since upgrading to Pixel 8

### DISCUSSION THREAD

**Discussion by dgor92** [BimmerGestalt/AAIdrive#778] [Q&A]
Hi,

Love the app - thank you! It worked perfectly with my last phone (Google Pixel 5) and I was really pleased with it. Using it for my 2020 Mini Clubman JCW.

I just upgraded to a Pixel 8 and for some reason AAIdrive now cannot connect to the car. I've tried re-installing AAIdrive and the Mini app. I've deleted and readded the car to the Mini app. I've checked both of them can run in the background and aren't optimised for battery saving etc. I've also tried adding the phone over Bluetooth using the Apps and Apple CarPlay options.

However, still when I turn on the car, the AAIdrive app continuously says "waiting for connection to car". It definitely says it's otherwise ready to connect to Mini. The Mini connects over Bluetooth for my phone for phone and audio, but doesn't show an option for Apps.

Have I done something wrong since changing phones or is this maybe a compatibility issue?

**Comment by hufman** [comment]
Thank you for the compliments!
Unfortunately, AAIdrive has little influence on the connection process from MyBMW/Mini to the car. Since you've already tried allowing background execution, and trying to connect to the car while the Mini app is open, I don't know what else to recommend.

**Reply by Krakuanen** [reply]
Same situation for me, no app option in the car menu since uppgrading to Motorola g84 (android 14).

**Comment by EvilDorf** [comment]
Had same problem on my new pixel 7, i forced Bluetooth 1.3 in dev menu, and gave all permissions to mybmw app and it started to give my phone apps option.

**Reply by Krakuanen** [reply]
I tried your settings, unfortunately it didn't work though. Still no apps in the menu. I wonder why...
If I connect via USB I get the apps so that's a solution.

**Reply by hallo1970** [reply]
No problems with my Pixel Pro 7 so far

**Comment by dgor92** [comment]
Hello all. Thanks for the comments so far! 

Things have recently started working for me in the car. No idea what I have done differently. I think I had to reset my WiFi and Bluetooth settings at some point for unrelated reasons, could that have been it?
